### PR TITLE
0001 -- Long Live The ODAO

### PR DESCRIPTION
**Pull request by zygomeb**
This is the official proposal to jumpstart ODAO.

Including the key dates and numbers

[Rendered](https://github.com/OptimFinance/odao-proposals/blob/proposal0001/0001-LongLiveTheODAO/0001-LongLiveTheODAO.md)

### CODE REVIEW

**Comment by Marco112358** [comment]
2 comments.

1. This section below, can we also propose the opposite? If the 1st stage is under subscribed and the 2nd stage is over subscribed, some OPTIzm from the 1st can be moved to the 2nd?

"Any available unclaimed OPTIM from the second swap phase will be allocated to the pool of available OPTIM for first swap phase participants. If the first swap phase is oversubscribed, any excess second phase OPTIM will be automatically added to fulfill orders placed during the first phase. It will be distributed on the same pro-rata basis as a participant’s percentage of the total order pool."

2. I think a necessary 2nd proposal will be establishing the ODAO and a compensation structure for members of the branches. Its important to attract talent to the ODAO and keep them incentivized

**Comment by zygomeb** [comment]
@Marco112358 

1. Yes, absolutely. It was the intention, in fact for it to work this way. Must have been lost in translation between the documents.

2. Definitely! We need some time to breathe to establish this properly and are definitely in no rush to push for expansion of talent pool. In so far as my opinion here is to 'signal intent' to push for it in the *vaguely near-to-midterm future* for more of the project be pushed into the open. It'll take some time and ultimately I think will be centered around the Optim Account, as the first proper Open Sourced component. Will be take a bit to move those cogs forward.

**Comment by Kazuyac** [comment]
I think , I assumed that all OPTIMBonds will be going for Phase 2. What I want to confirm is did I qualify extra OPTIMB for the cut-off on the 18/10 as feeding into Phase 2. This will decide how much I allocated to Phase 1. So, will there be a calculator of sorts to confirm prior to ILE?

**Comment by zygomeb** [comment]
@Kazuyac can you rephrase the question? Not sure I understand it

**Comment by Marco112358** [comment]
> @Kazuyac can you rephrase the question? Not sure I understand it

I believe he is asking is there a way to prove that the bonds u hold in ur wallet are approved for stage 2 (i.e. they were issued before the cut off date of 10/18). 

Will there be a UI that we can check to see what bonds in our wallets qualify. 

I think that's the question.

**Comment by zygomeb** [comment]
Right well in that case we didn't plan for such a UI be present but we definitely can add a preview on stage 2

**Comment by zygomeb** [comment]
Accepted in https://dao.optim.finance/#/proposal#31c172b45a8ed71beb609c581d1d93974ac27e1bbe0f8d4432b0f10d59885a1e
